app/components/ConnectorList.jsx

```diff
@@ -13,7 +13,7 @@ const ConnectorList = ({ t, connectors, showConnectedBadge = true }) => (
         iconName={c.slug}
         slug={c.slug}
         enableDefaultIcon
-        backgroundCSS={c.color.css}
+        backgroundCSS=''
```

Comment:
This connector list is not used for the discovery panel, so I can confirm that doesn't break this part.